common-libs/internal-api-clients/src/services/AffinidiVaultApiService.ts

```diff
@@ -11,15 +12,15 @@ export type BlobType = {
   id: number
 }
 
-type ApiSpec = typeof affinidiVaultSpec
+type ApiType = BuildApiType<ParseSpec<typeof affinidiVaultSpec>>
 
 @profile()
-export default class AffinidiVaultApiService extends GenericApiService<ApiSpec> {
+export default class AffinidiVaultApiService extends GenericApiService<ApiType> {
   constructor(options: ConstructorOptions) {
     super(options.vaultUrl, options, affinidiVaultSpec)
   }
 
-  async createDidAuthRequest(params: ExtractRequestType<ApiSpec, 'CreateDidAuthRequest'>) {
+  async createDidAuthRequest(params: ApiType['CreateDidAuthRequest']['requestBody']) {
```

Comment:
 IDE ask me to provide return type on function and I think we should do something like that 
 `async createDidAuthRequest(
    params: ApiType['CreateDidAuthRequest']['requestBody'],
  ): Promise<{ body: ApiType['CreateDidAuthRequest']['responseBody']; status: number }> {
    return this.execute('CreateDidAuthRequest', { params })
  }`
@inga-affinidi What do you think?

Reply:
@oleksiipiliugin , it's not IDE, it's our ESLint rules, and I don't think we need this specific rule (which caused to many cases of `: any` because instead of letting TS automatically infer things people had to declare something manually, and `any` is the easiest thing to declare)

Reply:
And I had to spend too much time on this issue already (to make parsers more or less universal), so honestly I'm not into doing any polishing right now.